server/models/Transaction.js

```diff
@@ -575,7 +575,7 @@ function defineModel() {
     return Promise.mapSeries(transactions, t => Transaction.create(t, opts)).then(results => results[index]);
   };
 
-  Transaction.createFeesOnTopTransaction = async ({ transaction, host }) => {
+  Transaction.createPlatformTipTransaction = async ({ transaction, host }) => {
```

Comment:
In [my PR](https://github.com/opencollective/opencollective-api/pull/5966/files#diff-4cffa0c51e597183bea1e391ac7dde45b32da7acdd7a8cfc4d2c42443d704097R609), I called this one `createPlatformTipTransactions` (with a `s`) since we're creating a transaction pair, but also the debt transactions

Reply:
Ok, updated to plural